src/copy_template.sh

```diff
@@ -1,4 +1,14 @@
 #!/bin/bash -
 
-cp /Dockerfile.template ${INPUT_TARGET_DIRECTORY}/Dockerfile
+if [ -z $CUSTOM_BASE_IMAGE_ALLOWED ] || [ -z $INPUT_BASE_IMAGE ]; then
```

Comment:
As long as this is mainly a feature flag rather than a security measure it seems fine.

Reply:
yeah, it should be okay that folks can work around this.